gramjs/client/TelegramClient.js

```diff
@@ -61,11 +61,16 @@ class TelegramClient {
         this.apiHash = apiHash
         this._useIPV6 = args.useIPV6
         this._entityCache = new Set()
-        if (typeof args.baseLogger == 'string') {
-            this._log = new Logger()
-        } else {
+
+        if (typeof(args.baseLogger) == 'function') {
```

Comment:
typeof is an operator, not a function in JavaScript.

Reply:
Ahh right